### ISSUE TITLE
Rllib-Multiagent error: TraCIException

### ISSUE THREAD

**Issue by KidTrunks** [flow-project/flow#702]
**## Bug Description**

I am trying to run the Rllib-multiagent script: multiagent_stabilizing_the_ring.py
I have tried to run the script multiple times, and every time the simulation terminates at the 9th iteration with error processing event. The number of training iterations is set to 250. 

**## Bug Reproduce**

**File in which the bug occurred:** /flow/examples/rllib/multiagent_exps/multiagent_stabilizing_the_ring.py

**The exact command that you did run that caused the bug:**
python multiagent_stabilizing_the_ring.py

**## Output Log**
The simulation always terminates at the 9th iteration with the following error codes:

traci.exceptions.TraCIException: Departure time in the past
traci.exceptions.TraCIException: Vehicle 'human_0_0' is not known

**## Screenshots**

![1 1](https://user-images.githubusercontent.com/45654203/63660964-16388680-c7fc-11e9-9059-36f4e4e86062.PNG)
![1 2](https://user-images.githubusercontent.com/45654203/63660966-189ae080-c7fc-11e9-97e9-68b5b89544b2.PNG)
![1 3](https://user-images.githubusercontent.com/45654203/63660967-1afd3a80-c7fc-11e9-91bf-fc3a2e8ab7ce.PNG)
![1 4](https://user-images.githubusercontent.com/45654203/63660970-1cc6fe00-c7fc-11e9-94b5-3190ad893a43.PNG)

**## Versions**:

**OS:** Ubuntu 18.04
**Sumo Version:** b'v0_31_0-812-g1d4338ab80'
**Ray version:** 0.6.4
 **Flow version:** 0.2.1

Any help would be greatly appreciated. Sorry about the trouble.

**Comment by eugenevinitsky** [comment]
Hi @KidTrunks, I suspect your SUMO version is the problem. If you upgrade to the new FLOW and use the new script there to install SUMO, that should do the trick.

**Comment by KidTrunks** [comment]
Hi @eugenevinitsky could you please link me the upgrade instructions to the new FLOW and SUMO version? Sorry I am unable to find the instructions.

**Comment by eugenevinitsky** [comment]
Hi @KidTrunks, you would just merge in the version that is on master and re-run the setup instructions for installing SUMO. Hope that helps!

**Comment by KidTrunks** [comment]
Thanks @eugenevinitsky the TraCIException was solved after updating FLOW and SUMO to the latest versions. 

However, a different error occurred, preventing SUMO simulation. 

**The file in which the bug occured is still:** /flow/examples/rllib/multiagent_exps/multiagent_stabilizing_the_ring.py (No changes are made to the master file)

**Output log:** ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()

![ValueError](https://user-images.githubusercontent.com/45654203/64058530-d9033880-cbef-11e9-9d45-39d749182c14.PNG)

I noticed that this is very similiar to #673 , but I am not sure where to check rl_action_list ?

Thanks in advance!

**Comment by KidTrunks** [comment]
I

**Comment by KidTrunks** [comment]
Hi @eugenevinitsky  I followed your advice and updated FLOW and SUMO to the latest versions. The previous issue is resolved, but a new issue appeared. 

I am still running the same script using the same command.

**Bug Output:** 
traci.exceptions.TraCIException: Invalid departlane definition for vehicle 'human_0_0';
must be one of ("random", "free", "allowed", "best", "first", or an int>=0

and,

traci.exceptions.TraCIException: Vehicle 'human_0_0' is not known

![2 1](https://user-images.githubusercontent.com/45654203/64395911-2fb0bc80-d09f-11e9-8fc7-428eea127f6e.PNG)
![2 2](https://user-images.githubusercontent.com/45654203/64395914-350e0700-d09f-11e9-9622-8d9199002c88.PNG)

**Versions:**
FLOW: 0.4.1 dev
SUMO: v1_1_0+0000-2147d155b1

I noticed that this bug is the same as issue #556 ? 
According to the post, it seems like it is a problem relating to the newest version of FLOW and was solved after down-grading FLOW. This doesn't make sense because I had TraCIException error coming from 0.2.0 as well ?

Any help to solving this issue would be greatly appreciated. Thanks.
